src/components/molecules/FormFields/AssetSelection.tsx

```diff
@@ -26,9 +26,9 @@ export default function AssetSelection({
           <input
             id={slugify(asset.name)}
```

Comment:
this is leading to a bug I had put in, when asset names are the same in the list. It's fixed in start compute branch and we should also add the fix here so we do not run into that bug when testing this branch. So just switch here `asset.name` to `asset.did`